apl-api2/src/main/resources/yaml/apollo-api-v2.yaml

```diff
@@ -623,587 +626,1649 @@ paths:
+          required: false
+          schema:
+            type: integer
+            format: int64
+            example: 1591696372000
+        - name: transaction
+          in: query
+          description: The transaction that contains contract
+          required: false
+          schema:
+            type: string
+            format: int64
+            example: 387615916963 or 0x34a11bc0012
+        - name: status
+          in: query
+          description: The contract status
+          required: false
+          schema:
+            type: string
+            description: the contract status ACTIVE, CLOSED, DESTROYED
+            example: ACTIVE
+        - $ref: '#/components/parameters/FirstIndexParameter'
+        - $ref: '#/components/parameters/LastIndexParameter'
+
+      responses:
+        200:
+          description: successful operation
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ContractListResponse'
+        400:
+          $ref: '#/components/responses/BadRequest'
+        500:
+          $ref: '#/components/responses/ServerError'
+
+  /@@smc/{address}:
+    get:
+      tags:
+        - smc
+      summary: Get detail contract information given address.
+      description: Returns detail contract information.
+      operationId: getSmcByAddress
+      parameters:
+        - name: address
+          in: path
+          description: The contract address
+          required: true
+          schema:
+            type: string
+            example: 'APL-X5JH-TJKJ-DVGC-5T2V8'
+      responses:
+        200:
+          description: successful operation
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ContractListResponse'
+        400:
+          $ref: '#/components/responses/BadRequest'
+        500:
+          $ref: '#/components/responses/ServerError'
+
+  /@@smc/{address}/event:
+    post:
+      tags:
+        - smc
+      summary: Get the contract events fired in the past.
+      description: Returns the contract events fired in the past.
+      operationId: getSmcEvents
+      parameters:
+        - name: address
+          in: path
+          description: The contract address
+          required: true
+          schema:
+            type: string
+            example: 'APL-X5JH-TJKJ-DVGC-5T2V8'
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/ContractEventsRequest'
+      responses:
+        200:
+          description: successful operation
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ContractEventsResponse'
+        400:
+          $ref: '#/components/responses/BadRequest'
+        500:
+          $ref: '#/components/responses/ServerError'
+
+  /@@smc/{address}/state:
```

Comment:
This request is paginated or secured somehow to restrict large state json retrieval?

Reply:
State it's the single json-object;